content/en/blog/2025/welcome-ratify-joined-notaryproject.md

```diff
@@ -34,8 +34,8 @@ These Ratify repositories have been transferred to Notary Project organization:
 
 By welcoming Ratify as an official subproject, the Notary Project now offers a broader and more opinionated solution stack for securing software supply chain:
 
-- Notation enables signing of OCI artifacts at the build stage and CI/CD.
-- Ratify enforces signature and other supply chain metadata verification policies by working with Gatekeeper in runtime and Kubernetes.
+- Notation enables signing of OCI artifacts in CI/CD pipelines.
+- Ratify enforces signature and other supply chain metadata verification policies by working with Gatekeeper in runtime, Kubernetes, and CI/CD.
```

Comment:
```suggestion
- Ratify enforces signature and other supply chain metadata verification policies by working with Gatekeeper on Kubernetes, and within CI/CD pipelines.
```

Reply:
@yizha1 ACK. I removed "Gatekeeper" and updated wording to make it clearer and simpler.